audit/projects/k8s-infra-prow-build-trusted/secrets/snyk-token/versions.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "createTime": "2021-06-15T20:14:58.824069Z",
+    "etag": "\"15c4d3a0d22785\"",
+    "name": "projects/180382678033/secrets/snyk-token/versions/4",
```

Comment:
Expected, this is iteration to get the value right for https://github.com/kubernetes/test-infra/pull/22570